python/auto-instrumentation/README.md

```diff
@@ -0,0 +1,81 @@
+#Steps to auto-instrument Python app
```

Comment:
To get the heading to render properly you need a space after the `#`

Reply:
Done!